learn/getting_started/customizing_relevancy.md

```diff
@@ -132,4 +132,21 @@ This will set `winnie` and `piglet` as synonyms; searching for either word will
 
 You can read more about it in our [dedicated guide](/learn/configuration/synonyms.md).
 
-The next chapter tackles more advanced topics including security and data backup.
+## Typo tolerance
+
+Meilisearch is typo tolerant by default. It will help you find relevant search results even if you make spelling mistakes or typos, e.g., searching for `swaj` instead of `swan`. The `typoTolerance` object allows you to:
+
+- Enable or disable the typo tolerance feature
+- Configure the minimum word size for typos
+- Disable typos on specific words
+- Disable typos on specific attributes
+
+Meilisearch accepts one typo for query terms containing `5` or more characters by default. If you search the movies index for `swaj`, you will not get any results.
```

Comment:
Should we be referring to 5 as `5`? On a quick read I briefly thought that `5` was a search term.

Reply:
I used `5` here because the [typo tolerance guide](https://docs.meilisearch.com/learn/configuration/typo_tolerance.htm) mostly uses backticks for number of characters